flink-table/flink-sql-client/src/test/resources/sql/function.q

```diff
@@ -1,4 +1,4 @@
-# function.q - CREATE/DROP/ALTER FUNCTION
+# function.q - CREATE/DROP/ALTER/SHOW/DESCRIBE FUNCTION
```

Comment:
```suggestion
# function.q - CREATE/DROP/ALTER/DESCRIBE FUNCTION
```

Reply:
why not include `SHOW`?

Reply:
Is there a SHOW FUNCTION?  Or is it SHOW FUNCTIONS?

Reply:
it is SHOW FUNCTIONS plural, but there are tests in this file that cover SHOW. This is also in line with the descriptions of table.q and model.q